### DISCUSSION TITLE
Is there any tutorial about how to install Forge over cloud ex. Vast.ai or Runpod?

### DISCUSSION THREAD

**Discussion by vishaldwdi** [lllyasviel/stable-diffusion-webui-forge#423] [General]
I recently learned about SD and its webUI, and I've come across some recent discussions about Forge and its efficiency. I must admit, I'm really impressed. The advancements made by Forge and its contributors are truly remarkable. Currently, I'm exploring the possibility of using SD for a pilot phase in my business vertical, which necessitates running SD WebUI on the cloud for distributed management. There are numerous tutorials available on running it on the cloud. However, after discovering the capabilities of Forge and being impressed by them, I now want to run it on the cloud as well. Unfortunately, there seems to be a lack of tutorials on installing Forge on the cloud. Therefore, I've turned to this community for assistance.

Can someone be help in providing tutorial about how to install Forge over Cloud. 
PS: i'm not interested in Google Colab since I got to know that Vast.ai would be much cheaper.

**Comment by jsonkcli** [comment]
You can run launch.py with the argument --share. 

duplicate of #189

> I recently learned about SD and its webUI, and I've come across some recent discussions about Forge and its efficiency. I must admit, I'm really impressed. The advancements made by Forge and its contributors are truly remarkable. Currently, I'm exploring the possibility of using SD for a pilot phase in my business vertical, which necessitates running SD WebUI on the cloud for distributed management. There are numerous tutorials available on running it on the cloud. However, after discovering the capabilities of Forge and being impressed by them, I now want to run it on the cloud as well. Unfortunately, there seems to be a lack of tutorials on installing Forge on the cloud. Therefore, I've turned to this community for assistance.
> 
> Can someone be help in providing tutorial about how to install Forge over Cloud. PS: i'm not interested in Google Colab since I got to know that Vast.ai would be much cheaper.

You can run launch.py with the argument --share. 

duplicate of #189
